pyproject.toml

```diff
@@ -76,29 +75,18 @@ test = [
 "Issues" = "https://github.com/edgi-govdata-archiving/wayback/issues"
 
 [build-system]
-requires = ["setuptools>=61.0", "versioneer[toml]==0.29"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling ~=1.19.1", "hatch-vcs ~=0.4.0"]
+build-backend = "hatchling.build"
 
-[tool.setuptools.packages.find]
-where = ["."]
-exclude = ["docs", "tests"]
+[tool.hatch]
+version.source = "vcs"
+build.hooks.vcs.version-file = "wayback/_version.py"
 
-[tool.setuptools.package-data]
-wayback = [
-    # When adding files here, remember to update MANIFEST.in as well, or else
-    # they will not be included in the distribution on PyPI!
-    # Details: https://setuptools.pypa.io/en/latest/userguide/datafiles.html
-]
-
-[tool.setuptools.dynamic]
-version = {attr = "wayback.__version__"}
+[tool.hatch.build.targets.sdist]
+exclude = [".git_archival.txt"]
 
-[tool.versioneer]
-VCS = "git"
-style = "pep440-post"
-versionfile_source = "wayback/_version.py"
-versionfile_build = "wayback/_version.py"
-tag_prefix = "v"
+[tool.hatch.build.targets.wheel]
+exclude = ["wayback/tests/*"]
```

Comment:
If you haven’t looked into it already, I suspect the build failures are because this is overriding the defaults, and ceasing to exclude some files that were excluded before: https://hatch.pypa.io/latest/plugins/builder/wheel/#default-file-selection